### ISSUE TITLE
table of contents for ‘long articles’

### ISSUE THREAD

**Issue by matthew-andrews** [Financial-Times/next-grumman#167]
initially where ‘long’ is defined as something sensible (my suggestion is maybe more than 2 subheadings?)

dependent on #163.

i'm not a fan of doing things by word count.  too magic/ropey imho.

**Comment by matthew-andrews** [comment]
this could go behind a feature flag

**Comment by matthew-andrews** [comment]
this ticket has superseded by Whiteboard™
